about/dds_overview.html

```diff
@@ -10,7 +10,7 @@
     <span class="headertext">Overview of DDS - DCPS</span>
 
     <p>Section 7.1.1.2 in the OMG's 1.2 DDS specification
-      <a href="http://www.omg.org/cgi-bin/doc?formal/07-01-01">(formal/07-01-01)</a> contains a more
+      <a href="https://www.omg.org/spec/DDS/1.2/About-DDS/">(formal/19-06-01)</a> contains a more
```

Comment:
The current spec (version 1.4) also has an overview section (2.2.1.2). I think we should use a link to the current version for any updated content.

Reply:
Updated to 1.4